modules/ROOT/pages/server-compatibility-transactions.adoc

```diff
@@ -3,4 +3,44 @@
 :idprefix:
 :idseparator: -
 
-include::partial$block-related-content-sync.adoc[]
+
+include::partial$_std-hdr-sgw.adoc[]
+
+
+// BEGIN -- Page Attributes
+:url-api-method-database-in-batch: http://docs.couchbase.com/mobile/{version-full}/couchbase-lite-swift/Classes/Database.html#/s:18CouchbaseLiteSwift8DatabaseC7inBatch5usingyyyKXE_tKF[Database.inBatch()]
+// END -- Page Attributes
+
+
+// BEGIN -- Page Heading
+:param-topic-group: compatibility
+:param-related: {compatibility--xref}
+include::partial$block-abstract.adoc[]
+// END -- Page Heading
+
+
+== Introduction
+
+Couchbase Server provides the backing data store for Sync Gateway and it is compatible -- see: {compatibility--xref}.
+However, there are considerations you needs to be aware of when using some Couchbase Server features with Sync Gateway.
+One of those instances concerns the use of Couchbase Server Transactions.
+
+
+== Local Transactions
+
+Within the Couchbase Mobile cluster all transactions are local, either to the Couchbase Server or to the Couchbase Lite database:
+
+* Couchbase Server supports server side transactions -- see: {server-transactions--xref}. +
+Changes made on the server within a transaction block are guaranteed to be imported by the Sync Gateway only after all the changes in a transaction are committed.
+
+* Similarly, Couchbase Lite supports transactions that are local to the Couchbase Lite database -- see: {url-api-method-database-in-batch}. +
+Only committed transactions will ever reach Sync Gateway.
+
+Sync Gateway is not transaction-aware.
+There is no guarantee that a series of updates made within a transaction block, on either the Couchbase Server or Couchbase Lite client side, will be ACID compliant when they sync to the other end.
+
+If there is a use-case to build end-to-end transactions between Couchbase Server and Couchbase Lite clients, you would implement it within the application layer.
+Perhaps by having a separate _control document_ of transaction log entries, which must be synced from the clients that capture the list of related changes.
```

Comment:
I don't know about proposing this hypothetical approach - wouldn't it be better to limit the docs to describing the functionality we do support?

Reply:
removed